R/update.R

```diff
@@ -207,9 +211,9 @@ pkg_source <- function (desc)  {
 
 .get_version <- function(x) {
   url_con <- url(x)
-  res <- as.character(read.dcf(url_con, fields="Version"))
-  close(url_con)
-  res
+  on.exit(close(url_con))
```

Comment:
@SridharJagannathan this pattern of using `on.exit` to close a connection is a standard approach to deal with the possibility of an error during a read/write operation. It's important to close connection objects as they are a limited OS resource.